src/main/java/gyro/google/storage/BucketBilling.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.storage;
+
+import com.google.api.services.storage.model.Bucket.Billing;
+import gyro.core.resource.Diffable;
+import gyro.core.resource.Updatable;
+import gyro.google.Copyable;
+
+/**
+ * Subresource for setting the Bucket Billing configuration to a Bucket.
+ */
+public class BucketBilling extends Diffable implements Copyable<Billing> {
+
+    private Boolean requesterPays;
+
+    /**
+     * When ``true`` the requester pays setting for this bucket.
+     */
+    @Updatable
+    public Boolean getRequesterPays() {
+        return requesterPays;
+    }
+
+    public void setRequesterPays(Boolean requesterPays) {
+        this.requesterPays = requesterPays;
+    }
+
+    @Override
+    public String primaryKey() {
+        return Boolean.toString("true".equals(getRequesterPays()));
```

Comment:
I'd return `""` for this so the output looks a bit nicer.

Reply:
Done.